variation/to_vrs.py

```diff
@@ -201,23 +105,57 @@ async def to_vrs(self, q: str,
             unable to translate or normalize query.
         :return: ToVRSService containing VRS variations and warnings
         """
-        validations, warnings = await self.get_validations(q)
-        translations, warnings = self.get_translations(validations, warnings)
+        warnings = []
+        variations = []
+        params = {
+            "search_term": q,
+            "variations": variations,
+            "service_meta_": ServiceMeta(
+                version=__version__, response_datetime=datetime.now()
+            ),
+            "warnings": warnings,
+        }
+
+        # Get tokens for input query
+        tokens = self.tokenizer.perform(unquote(q.strip()), warnings)
+        if warnings:
+            params["warnings"] = warnings
+            return ToVRSService(**params)
+
+        # Get classification for list of tokens
+        classification = self.classifier.perform(tokens)
```

Comment:
If `tokens` only has to be an Iterable, this is technically a type violation since `Classifier.perform` expects a List

Reply:
I forgot to update return types for perform methods. Will look at others